src/main/scala/satify/dsl/DSL.scala

```diff
@@ -11,6 +12,17 @@ object DSL:
     def apply(s: String): Symbol = Symbol(s)
 
   extension (exp1: Expression)
+
     def and(exp2: Expression): Expression = And(exp1, exp2)
     def or(exp2: Expression): Expression = Or(exp1, exp2)
     def not: Expression = Not(exp1)
+    def xor(exp2: Expression): Expression = Or(And(exp1, Not(exp2)), And(Not(exp1), exp2))
+
+    @targetName("andSymbol")
+    def /\(exp2: Expression): Expression = and(exp2)
+    @targetName("orSymbol")
+    def \/(exp2: Expression): Expression = or(exp2)
+    @targetName("notSymbol")
+    def unary_! : Expression = not
+    @targetName("xorSymbol")
+    def ⊕(exp2: Expression): Expression = xor(exp2)
```

Comment:
I think the Xor symbol is not so easy for a user to write, what do you think of '^', or just a 'X' ?